enclave_build/src/image_manager.rs

```diff
@@ -0,0 +1,251 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+#![allow(dead_code)]
+
+use tempfile::NamedTempFile;
+
+use crate::image::ImageDetails;
+use crate::storage::OciStorage;
+use crate::{EnclaveBuildError, Result};
+
+/// Trait which provides an interface for handling images.
```

Comment:
Nit: maybe this could be rephrased to "handling a particular image". The first time I read it I thought it would manage a collection of images.